### PR TITLE
feat: emojicode

### PR DESCRIPTION
**Pull request by ThePrimeagen**

### CODE REVIEW

**Review by joshmanders — APPROVED** [review]
LGTM 👍

**Review by gabrielrbarbosa — COMMENTED** [review]
👍

**Review by HackTuah — COMMENTED** [review]
🍆

**Review by EverStarck — APPROVED** [review]
LGTM!

File: loops/emojicode/code.emojic [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    🆕🎰❗️➡️ rand
+    🔢rand 0 9999❗️➡️r
+
+	🆕🍨🐚🔢🍆❗️ ➡ 🖍🆕a
+    🔂 i 🆕⏩ 0 10000❗️🍇
+        🐻 a 0❗️
+    🍉
+
+    🔂 i 🆕⏩ 0 10000❗️🍇
+        🔂 j 🆕⏩ 0 10000❗️🍇
```
Comment (bddicken) [inline loops/emojicode/code.emojic:13]:
```suggestion
        🔂 j 🆕⏩ 0 100000❗️🍇
```

**Comment by bddicken** [comment]
Run on battery which negatively impacts performance, however:

```
Benchmarking Emojicode
Benchmark 1: ./emojicode/code 40
  Time (mean ± σ):     92.406 s ±  0.543 s    [User: 91.077 s, System: 0.298 s]
  Range (min … max):   91.851 s … 92.936 s    3 runs
```

**Review by rodrigo462003 — APPROVED** [review]
LGTM

**Review by mikkurogue — APPROVED** [review]

**Review by leeozaka — APPROVED** [review]

**Comment by ThePrimeagen** [comment]
@bddicken your change approved

File: run.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ run "Crystal" "./crystal/code 40"
 #run "Ada" "./ada/code 40"
 #run "D" "./d/code 40" # Seems to not have an arm / M1 version
 run "Odin" "./odin/code 40"
-run "Fortran" "./fortran/code 40" 
+run "Fortran" "./fortran/code 40"
```
Comment (Blackbox-MIT-dev) [inline run.sh:40]: Please remove unrelated formatting changes

**Comment by bddicken** [comment]
Production-ready code right here. Let's ship it.
